java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/catalog/OpenmldbCatalogService.scala

```diff
@@ -35,6 +37,29 @@ class OpenmldbCatalogService(val zkCluster: String, val zkPath: String) {
     throw new Exception("fail to create sql executor")
   }
 
-  // TODO: use sql router to get catalog data from NameServer
+  def getDatabases(): List[String] = {
+    val databases = new VectorString()
+    val status = new Status
+
+    sqlRouter.ShowDB(databases, status)
+    if (status.getCode == 0) {
+      databases.toList
+    } else {
+      null
+    }
```

Comment:
should delete `databases` and `status` manually after used.  refer (here)[https://github.com/4paradigm/OpenMLDB/blob/main/java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/impl/InsertPreparedStatementImpl.java#L57]

Reply:
May be we should add the methods in SqlClusterExecutor, then openmldb-batch can use SqlClusterExecutor, avoid to load jsdk libs and do delete swig objects here(let's swig types hide in SqlClusterExecutor).

Reply:
We are not extending `SqlClusterExecutor` in `openmldb-jdbc` since it is designed for JDBC API. But we have to handle the same issues as `SqlClusterExecutor`.